### PR TITLE
add usegalaxy.be wf and histories

### PR DESCRIPTION
**Pull request by ieguinoa**

### CODE REVIEW

**Comment by bgruening** [comment]
@ieguinoa can you also create history just with the datasets, e.g. https://usegalaxy.eu/u/wolfgang-maier/h/covid-19-raw-data this way you can at least process as well the pre-processing.

Ping me on gitter if you need anything.

File: 3-MRCA/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+| [![Galaxy workflow](https://img.shields.io/static/v1?label=workflow&message=run&color=blue)](https://usegalaxy.org/u/aun1/w/mrca) | [![Galaxy workflow](https://img.shields.io/static/v1?label=workflow&message=run&color=blue)](https://usegalaxy.eu/u/wolfgang-maier/w/covid-19-mrca-analysis) | [![Galaxy workflow](https://img.shields.io/static/v1?label=workflow&message=run&color=blue)](https://usegalaxy.org.au/u/simongladman/w/covid-19-mcra) | [![Galaxy workflow](https://img.shields.io/static/v1?label=workflow&message=run&color=blue)](https://usegalaxy.be/u/ieguinoa/w/covid-19-mcra) |
+| [![Galaxy history](https://img.shields.io/static/v1?label=history&message=view&color=blue)](https://usegalaxy.org/u/aun1/h/covid-19-mrca-date) | [![Galaxy history](https://img.shields.io/static/v1?label=history&message=view&color=blue)](https://usegalaxy.eu/u/wolfgang-maier/h/covid-19-mrca-analysis) | [![Galaxy history](https://img.shields.io/static/v1?label=history&message=view&color=blue)](https://usegalaxy.org.au/u/simongladman/h/covid-19-mcra) | [![Galaxy history](https://img.shields.io/static/v1?label=history&message=view&color=blue)](https://usegalaxy.be/u/ieguinoa/h/3mrca) |
```
Comment (bgruening) [inline 3-MRCA/README.md]: @ieguinoa can you rename your histories to include the prefix `covid-19-` and ideally also tag them, the histories and workflows?

Comment (ieguinoa) [reply 3-MRCA/README.md]: no problem

**Comment by ieguinoa** [comment]
this last commit contains all the workflows and histories with standardized names. They are also all tagged

**Comment by bgruening** [comment]
Sorry, there is a conflict now ... can you take care or should I?

**Comment by ieguinoa** [comment]
> Sorry, there is a conflict now ... can you take care or should I?

you, go I guess it's because of something you added in between :)

**Comment by bgruening** [comment]
Pff ;)  @ieguinoa your pre-processing history is still empty ... only RAW files in it.

**Comment by bgruening** [comment]
If you have time please double check all links.

**Comment by bgruening** [comment]
@ieguinoa please feel to also add the alternate workflow and histories here: https://github.com/galaxyproject/SARS-CoV-2/tree/master/1-PreProcessing#alternate-workflow

**Comment by ieguinoa** [comment]
> Pff ;) @ieguinoa your pre-processing history is still empty ... only RAW files in it.

running now



> @ieguinoa please feel to also add the alternate workflow and histories here: https://github.com/galaxyproject/SARS-CoV-2/tree/master/1-PreProcessing#alternate-workflow

just made a PR with those links

**Pull request merged**